zuul.d/jobs.yaml

```diff
@@ -478,3 +478,48 @@
     run: playbooks/packer-functional-devstack/run.yaml
     vars:
       packer_version: 1.2.5
+
+- job:
+    name: docker-machine-master-functional-opentelekomcloud
+    parent: init-test
+    description: |
+      Test functionalities of Docker machine of master against opentelekomcloud.
+    run: playbooks/docker-machine-functional-public-clouds/run.yaml
+    vars:
+      cloud_name: opentelekomcloud
+    secrets:
+      - opentelekomcloud_credentials
+
+- job:
+    name: docker-machine-maste-functional-orange
```

Comment:
s/maste/master/
And the following job names

Reply:
done